ADAR/donations.yml

```diff
@@ -5,12 +5,23 @@
     address: 986 Mission Street, 4th Floor, San Francisco 94103, California, USA
     tel: +1 828 479 5482
     contact: contactus@mail.kiva.org
-  ada: 16.13 # <- These must be numbers. Can be decimals though. See below
+  ada: 0.0 # <- These must be numbers. Can be decimals though. See below
```

Comment:
It would be good to leave it as it was. Even if the donation wasn't made in ADA at least we can have a total of equivalent ADA donated.

Reply:
Updated